web/src/components/navbar/NavBarDesktop.tsx

```diff
@@ -26,12 +26,19 @@ export const NavBarDesktop = (): ReactElement => {
   const anchorRef = useRef<HTMLButtonElement | null>(null);
 
   const toggleDropdown = (): void => {
+    if (currentlyOnboarding()) {
+      return;
+    }
```

Comment:
this is unnecessary, right? Because the dropdown will never even show up when in onboarding?